TextToSpeech/README.md

```diff
@@ -0,0 +1,67 @@
+# Cloud Text-to-Speech for PHP
+
+> Idiomatic PHP client for [Cloud Text-to-Speech](https://cloud.google.com/text-to-speech).
+
+[![Latest Stable Version](https://poser.pugx.org/google/cloud-text-to-speech/v/stable)](https://packagist.org/packages/google/cloud-text-to-speech) [![Packagist](https://img.shields.io/packagist/dm/google/cloud-text-to-speech.svg)](https://packagist.org/packages/google/cloud-text-to-speech)
+
+* [API documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/cloud-text-to-speech/latest/texttospeech/readme)
+
+**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googlecloudplatform/google-cloud-php). Any
+support requests, bug reports, or development contributions should be directed to
+that project.
+
+### Installation
+
+To begin, install the preferred dependency manager for PHP, [Composer](https://getcomposer.org/).
+
+Now to install just this component:
+
+```sh
+$ composer require google/cloud-text-to-speech
+```
+
+Or to install the entire suite of components at once:
+
+```sh
+$ composer require google/cloud
+```
+
+
```

Comment:
We may need to fix this in the generation tool. There is an extra space here.

Reply:
Done